src/systemd/cockpit.service.in

```diff
@@ -11,8 +11,6 @@ After=cockpit-wsinstance-http.socket cockpit-wsinstance-https-factory.socket
 
 [Service]
 RuntimeDirectory=cockpit/tls
-# systemd ≥ 241 sets this automatically
-Environment=RUNTIME_DIRECTORY=/run/cockpit/tls
```

Comment:
Since you asked on chat: RHEL 9 has systemd 252, debian-stable has 252, ubuntu 22.04 has 249. So this should be fine. Also, our tests would completely explode if that env wasn't set, so: happy bots, happy pitti :wink: